### PR TITLE
Override accessor to provide fallback; have controllers do validation

### PR DESCRIPTION
**Pull request by cjcolvar**
This ended up getting a little ugly but the tests pass.

### CODE REVIEW

File: app/models/access_control_step.rb [hunk 1]
```diff
@@ -94,13 +94,22 @@ def execute context
       media_object.governing_policies.delete( lease )
       lease.destroy
     end
+    #if context['add_lending_period_days'].present? || context['add_lending_period_hours'].present?
+    #  lending_period = build_lending_period(context['add_lending_period_days'], context['add_lending_period_hours'])
+    #  media_object.lending_period = lending_period if lending_period.positive?
+    #end
+
```
Comment (masaball) [inline app/models/access_control_step.rb:101]: Should this commented out code be removed?

**Review by masaball — APPROVED** [review]
Looks good.

File: app/models/access_control_step.rb (hunk 1 above)
Comment (cjcolvar) [reply app/models/access_control_step.rb:101]: Yes, good catch.

**Review by masaball — APPROVED** [review]
Looks good

**Pull request merged**